OptimizelySDK/Event/Entity/SnapshotEvent.cs

```diff
@@ -36,12 +39,12 @@ public class SnapshotEvent
         public int? Revenue { get; private set; }
 
         [JsonProperty("value", NullValueHandling = NullValueHandling.Ignore)]
-        public long? Value { get; private set; }
+        public float? Value { get; private set; }
 
         [JsonProperty("tags", NullValueHandling = NullValueHandling.Ignore)]
         public EventTags EventTags { get; private set; }
-
-        public SnapshotEvent(string entityId, string uuid, string key, long timestamp, int? revenue, long? value, EventTags eventTags)
+        
+        public SnapshotEvent(string entityId, string uuid, string key, long timestamp, int? revenue = null, float? value = null, EventTags eventTags = null)
```

Comment:
How come `SnapshotEvent` doesn't get a `Builder`?

Reply:
you are right, I will add builder for it, initially I decided not to have builder for simple classes but this class have same type of properties which may be interchanged. I actually missed it. Will add